client/components/presentation/Common/CommonModal.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+/**
+ * @class CommonModal
+ * @extends React.Component
+ */
+class CommonModal extends React.Component {
```

Comment:
CommonModal component has been removed. it was aimed at DRYing up the code but in the course of refactoring, I saw it did little to achieve that purpose so I let it go.